### PR TITLE
@onanna added the service layer and mock testing for createTie

### PR DESCRIPTION
**Pull request by onanna**

### CODE REVIEW

File: src/main/java/com/cottonfactory/products/controllers/TieController.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("api/products/ties")
+@Api(tags = {"ties"})
+public class TieController {
+
+    private TieService service;
+
+    @Autowired
```
Comment (psivakoti) [inline src/main/java/com/cottonfactory/products/controllers/TieController.java]: Autowired annotation can be removed from this.

**Comment by mosdevly** [comment]
Duplicated in #21

**Comment by onanna** [comment]
they are multiple tests, so cherry picked commits, into pr but because they have not been merged it is all showing up

**Comment by onanna** [comment]
21 got closed because this is a more up-to-date change with one  test case

**Comment by onanna** [comment]
> Duplicated in #21

resolved 22 contains more up to date changes

**Review by mosdevly — APPROVED** [review]

**Pull request merged**